modules/cnf-du-updating-crio.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-updating-crio_{context}"]
+
+= Updating CRI-O
+
+In support of this feature, CRI-O supports new configuration settings. These are applied by the MachineConfig specified above.
+
+----
+[crio.runtime.workloads.{workload-type}]
+  activation_annotation = "target.workload.openshift.io/{workload-type}"
+  annotation_prefix = "resources.workload.openshift.io"
+  resources = { "cpushares" = 0, "cpuset" = "0-1,52-53" }
+----
+
```

Comment:
The typical guidance in the doc guidelines for code blocks and explanations is to use <1> annotations and a callout list.  Scroll down a bit from [Code blocks](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output).

Unless there is a reason to use a mix of cpuset and CPU set, please use one for consistency.

If PerformanceProfile is an API object, please enclose in backticks and follow with "object" or "resource"--in accordance with doc guidelines for [object references](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-object-references).  Or, maybe just "performance profile".